diffpy/snmf/optimization.py

```diff
@@ -2,36 +2,39 @@
 import cvxpy as cp
```

Comment:
don't change the name on import in this case to make the code more readable later.   We do this for numpy and a few other things because that makes the code more readable becuase it is used so much, but not in this case.

Reply:
Ok. I will remove the name change